docs/csharp/indexers.md

```diff
@@ -72,12 +72,7 @@ Indexers enable you to provide the API that matches your type's abstraction
 without exposing the inner details of how the values for that abstraction
 are stored or computed.
 
-Let's walk through some of the common scenarios for using *indexers*.
-The code for all the samples is available at the [dotnet/docs GitHub
-repository](https://github.com/dotnet/docs). Otherwise, you can
-access the 
-[sample folder](https://github.com/dotnet/docs/tree/master/samples/csharp/indexers)
-directly.
+Let's walk through some of the common scenarios for using *indexers*. The code for all of the [samples](https://github.com/dotnet/docs/tree/master/samples) is available at the dotnet/docs GitHub repository. Otherwise, you can access the [sample folder for indexers](https://github.com/dotnet/docs/tree/master/samples/csharp/indexers) directly. For download instructions, see [Samples and Tutorials](../samples-and-tutorials/index.md#viewing-and-downloading-samples).
```

Comment:
I don't see the value of providing the location for all samples, followed by the location for the samples related to this topic. I'd remove the sentence beginning "The code for all..." And rephrase the following sentence (remove "Otherwise", remove "directly".